src/js/components/NavBar.js

```diff
@@ -112,7 +112,7 @@ class NavBar extends Component {
                   overlay={menu}
                   animation="slide-up"
                 >
-                  <Text>Chunk {this.props.chunkNum}</Text>
+                  <Text>{searchBar.mode} {this.props.chunkNum}</Text>

```

Comment:
if the mode changes, won't this.props.chunkNum have to change too?
Just curious?

Reply:
no, from the verse project that you sent me the only thing that it changes in the manifest file it is the mode, everything else remains the same. in this case we could rename the variable for something more generic like number. 

![screen shot 2018-04-16 at 4 38 36 pm](https://user-images.githubusercontent.com/25593318/38833859-ad1a9e48-4194-11e8-8480-f01bcb638bd4.png)

Reply:
ok cool. thanks @antonlopez